packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -558,6 +558,10 @@ export class MLSService extends TypedEventEmitter<Events> {
 
   public async handleEvent(params: Omit<EventHandlerParams, 'mlsService'>): EventHandlerResult {
     return handleBackendEvent({...params, mlsService: this}, async groupId => {
+      const conversationExists = await this.conversationExists(groupId);
+      if (!conversationExists) {
+        return;
+      }
```

Comment:
This is the main change, we make sure conversation still exist before checking epoch number and emitting an event. It's possible that upon decrypting an mls message (inside `handleBackendEvent`, see https://github.com/wireapp/wire-web-packages/blob/a68052626eea553fcd0b5142ac15b6ae92da5aff/packages/core/src/messagingProtocols/mls/EventHandler/events/messageAdd/messageAdd.ts#L60) we'll be removed from a group and it will be wiped.